ingest/source-data/ncbi-dataset-field-map.tsv

```diff
@@ -0,0 +1,17 @@
+key	value
+Accession	genbank_accession_rev
+Source database	database
+Isolate Lineage	strain
+Geographic Region	region
+Geographic Location	location
+Isolate Collection date	collected
+Release date	submitted
+Update date	updated
+Length	length
+Host Name	host
+Isolate Lineage source	isolation_source
+BioProjects	bioproject_accession
```

Comment:
```suggestion
BioProjects	bioproject_accessions
```

Reply:
Ended up dropping the `BioProjects` column entirely since it is not used in the build (https://github.com/nextstrain/dengue/commit/116f363aa8ddba449720bc6feba2f7212390f781). I'll keep this comment in mind if we add it back in.